Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/FeedbackForm.hs

```diff
@@ -0,0 +1,33 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Storage.Queries.FeedbackForm where
+
+import Domain.Types.FeedbackForm
+import Kernel.Prelude
+import Kernel.Storage.Esqueleto as Esq
+import Storage.Tabular.FeedbackForm
+
+findAllFeedback :: Transactionable m => m [FeedbackFormRes]
```

Comment:
Can't see any use of these queries.

Reply:
the types , queries and table of feedback form at BPP is not being used right now, but @RitikaHanish told me to add them for future use

Reply:
What kind of future use?

Reply:
@RitikaHanish